src/components/mermaidMap/Map.styles.js

```diff
@@ -17,16 +18,15 @@ export const MapWrapper = styled.div`
   min-height: ${(props) => (props.minHeight ? props.minHeight : '70vh')};
 
   #marker {
-    background-image: url('https://maplibre.org/maplibre-gl-js-docs/assets/custom_marker.png');
+    background-image: url(${MapPin});
     background-size: cover;
-    width: 35px;
-    height: 44px;
+    width: 31px;
+    height: 31px;
     cursor: move;
   }
 
   svg {
-    width: 3rem;
-    height: 3rem;
+    display: none;
```

Comment:
Interesting. Why did we need to supress an svg from being visible. Is that a hack? Is the styling for this SVG set somewhere else and we are undoing it here?

Reply:
here we added a custom marker png (via map-libre add marker class) instead of the built-in svg